<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainClientWindow</class>
 <widget class="QMainWindow" name="MainClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Чат Программа alpha release</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Чат</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_new_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Введите новое сообщение:</string>
    </property>
   </widget>
   <widget class="QListView" name="list_messages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Отправить </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="menu_clear"/>
    <addaction name="menu_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="menu_add_contact">
   <property name="text">
    <string>Добавить контакт</string>
   </property>
  </action>
  <action name="menu_del_contact">
   <property name="text">
    <string>Удалить контакт</string>
   </property>
  </action>
  <action name="menu_clear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
